cmc/values.preview.yaml

```diff
@@ -53,6 +53,7 @@ cmc-claim-store:
       PDF_SERVICE_URL: '{{ .Values.global.pdfServiceUrl }}'
       SEND_LETTER_URL: '{{ .Values.global.sendLetterUrl }}'
       DOCUMENT_MANAGEMENT_URL: '{{ .Values.global.documentManagementUrl }}'
+      DOC_ASSEMBLY_URL: '{{ .Values.global.dgAssemblyUrl }}'
```

Comment:
Shouldn't need this

Reply:
it using AAT endpoint ...

Reply:
You are overriding this line with same value. https://github.com/hmcts/chart-cmc/pull/43/files#diff-483f7a4b2394c948e242ab7122a8aa52R66
```suggestion
```

Reply:
done